resources/js/Pages/Tasks/Task.vue

```diff
@@ -40,9 +40,9 @@
 <template>
   <Layout>
     <div id='task'>
-      <h4 class='mb-5'>Task</h4>
+      <h4 class='mb-5'>Task #{{ task.id }}</h4>
       <section class='grid-col-3 mb-3'>
-        <p><b>Task ID:</b> {{ task.id }}</p>
+        <p><b>Title:</b> {{ task.title }}</p>
         <p><b>Author:</b> {{ task.author.name }}</p>
         <p><b>Created:</b> {{ new Date(task.created_at).toLocaleDateString() }}</p>
```

Comment:
@jlrusso do we plan on turning these into the ../shared/Input component later? That way they can be edittable?

Reply:
Yeah